jcp/parser.js

```diff
@@ -43,19 +43,19 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
       result.unitid = itemid[1];
     }
 
-    if (match[1] === 'JCP' || match[1] === 'jcp') {
+    if (match[1].toLowerCase() === 'jcp') {
       result.doi = jcp_doi_prefix + itemid[1];
-    } else if (match[1] === 'PCC' || match[1] === 'pcc') {
+    } else if (match[1].toLowerCase() === 'pcc') {
       result.doi = pcc_doi_prefix + itemid[1];
     }
   } else if ((match = /^\/([a-zA-Z]{3})\/article\/(Pages|pages)\/\d\d\d\d\/[a-z0-9]*\/(.*)\.aspx$/i.exec(path)) !== null) {
```

Comment:
1. ``/i`` allows to make the regex test case insensitive. it is not necessary to make ``(Pages|pages)``, ``pages`` is more than enough
2. Why do ``[a-zA-Z]{3}`` to get a year back when you do ``[a-zA-Z]{3}`` (ditto in this case the ``A-Z`` is useless with the ``/i``) do ``[0-9]{4}``